packages/sdk/src/waku/waku.ts

```diff
@@ -106,7 +106,16 @@ export class WakuNode implements IWaku {
     this.health = getHealthManager();
 
     if (protocolsEnabled.store) {
-      const store = wakuStore(this.connectionManager);
+      let peerIdStr: PeerIdStr | undefined;
```

Comment:
it is not useful variable as there is not value assignment to it between lines 109 and 119, so at 118 it is always `undefined`.

Reply:
good catch, updated with `options.store.peer`